userguide/storage.rst

```diff
@@ -1371,17 +1371,22 @@ changed, and destroying a zvol requires confirmation.
 Managing Encrypted Volumes
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-%brand% generates and stores a randomized *encryption key* whenever
+%brand% generates a randomized *encryption key* whenever
 a new encrypted volume is created. This key is required to read and
-decrypt any data on the volume.
-
-Encryption keys can also be downloaded as a safety measure, to allow
-decryption on a different system in the event of failure, or to allow
-the locally stored key to be deleted for extra security. Encryption
-keys can also be optionally protected with a *passphrase* for
-additional security. The combination of encryption key location and
-whether a passphrase is used provide several different security
-scenarios:
+decrypt any data on the volume. 
+
+By default, %brand% stores encryption keys locally within the system
+settings so volumes can automatically decrypt when the system
```

Comment:
"within the system settings" is not clear.  Does it mean "in the system configuration database"?

s/can automatically decrypt/can be automatically decrypted/

This would also be an opportunity to avoid starting a sentence with a halting aside by moving the "by default" to later:

`` %brand% stores encryption keys on the system boot pool so encrypted pools can be automatically decrypted when the system starts. ``

Reply:
Done